extension/source/ui/popup/popup.html

```diff
@@ -19,7 +19,7 @@
         Spoilers Arming
       </button>
     </div>
-    <div id="spoilerarmingholder">
+    <div id="keyboardmappingholder">
```

Comment:
This had the wrong id, it might have also caused a problem we weren't aware of

Reply:
Thanks for fixing that!